docs/src/modules/components/AppNavDrawer.js

```diff
@@ -231,7 +231,7 @@ function ProductDrawerButton(props) {
               href={ROUTES.dataGridDocs}
               // eslint-disable-next-line material-ui/no-hardcoded-labels
             >
-              Data Grid <KeyboardArrowRight fontSize="small" />
+              Advanced components <KeyboardArrowRight fontSize="small" />
```

Comment:
cc @flaviendelangle @DanailH @m4theushw

Reply:
It looks clean like this but we either need to change the link once we have the docs for the Date Picker to lead to a generic page for the MUI X.
Another option would be to have 2 links there, one for Data Grid and another for Date Picker, but this won't scale as if we add another component or 2 we need the additional links.
So option 1 would be better - single link that leads to a "Getting Started" page for MUI X.
cc @joserodolfofreitas

Reply:
Moved to #30972